### PR TITLE
Add info about _time and _sysTime

### PR DESCRIPTION
**Pull request by manototh**

### CODE REVIEW

**Comment by manototh** [comment]
Changelog: https://github.com/axiomhq/axiom-co/pull/1127

**Comment by dominicchapman** [comment]
I don’t think the sorting on projected story is now true. Let’s check in with DB on next steps here. Check out AXM-3868. We can take the additions to reference/datasets and the other cleanup though :-)

**Review by s-scheele-simonsen — APPROVED** [review]

**Comment by manototh** [comment]
@dominicchapman Thanks for this. I've clarified this with DB now. Could you please take another look and approve if happy?

File: apl/tabular-operators/sort-operator.mdx [hunk 1]
```diff
@@ -21,6 +21,18 @@ Sorts the rows of the dataset into an order by one or more columns.
 | asc        | **string**     | Sort by into ascending order, low to high.                                                    |
 | desc       | **string**     | Sort by into descending order, high to low.                                                   |
 
+## Use sort and project together
+
+When you use `project` and `sort` in the same query, ensure you project the fields that you want to sort on. Similarly, When you use `project-away` and `sort` in the same query, ensure you don’t remove the fields that you want to sort on.
```
Comment (manototh) [inline apl/tabular-operators/sort-operator.mdx]:
```suggestion
When you use `project` and `sort` in the same query, ensure you project the fields that you want to sort on. Similarly, when you use `project-away` and `sort` in the same query, ensure you don’t remove the fields that you want to sort on.
```

**Review by 54892 — APPROVED** [review]

**Review by 54892 — APPROVED** [review]

**Review by dominicchapman — APPROVED** [review]
LGTM

**Pull request merged**